spec/factories/pairings.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :pairing do
+    date_time { Faker::Time.between(from: DateTime.now - 365, to: DateTime.now + 365) }
+    user1_cancelled { [true, false].sample }
+    user2_cancelled { [true, false].sample }
+    association :user1, factory: :user
+    association :user2, factory: :user
```

Comment:
This is how you can alias associations in factory bot

Reply:
Oh nice so it will create a user within the factory